src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Color;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int COUNT_OF_FIGURES = 5;
+    private static final int MAX = 100;
+    private static final int DEFAULT_FIGURE_PROPERTY_VALUE = 10;
+    private static final Color DEFAULT_FIGURE_COLOR = Color.WHITE;
+    private Random magic = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    private int getProperValue() {
+        return magic.nextInt(MAX) + 1;
+    }
+
+    public Figure getRandomFigure() {
+        int typeOfFigure = magic.nextInt(COUNT_OF_FIGURES) + 1;
+
+        switch (typeOfFigure) {
+            case 1:
+                int radius = getProperValue();
+                return new Circle(colorSupplier.getRandomColor(), radius);
+
```

Comment:
lets create a separate method to get each random figure, this would make your code easier to read
```suggestion
            case 1:
                return getRandomCircle();
```